.gitignore

```diff
@@ -42,3 +42,5 @@ yarn-debug.log*
 /.yarnrc
 
 .DS_Store
+
+*.rdb
```

Comment:
No newline at the end of the file here. Also, I'm not sure adding dump.rdb needs to be part of this PR.